packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -129,22 +130,25 @@ contract SystemConfig is OwnableUpgradeable, ISemver {
     /// @notice The operator fee constant.
     uint64 public operatorFeeConstant;
 
+    /// @notice The L2 chain ID that this SystemConfig configures.
+    uint256 public l2ChainId;
```

Comment:
Now that this is accessible in the system config, OPCM should verify the L2 chain ID that's in the dispute game contracts matches this. Even better, remove the immutable from the dispute games and read it from here

Reply:
We can probably verify it somewhere but changing it to read from SystemConfig is too big of a diff for this set of changes